analyzer/src/main/kotlin/managers/GoMod.kt

```diff
@@ -292,7 +283,10 @@ class GoMod(
         }
     }
 
-    private fun ModuleInfo.toPackage(): Package {
+    private fun ModuleInfo.toPackage(): Package? {
+        if (version.isBlank()) return null // A ModuleInfo with blank version should be represented by a Project.
+        if (replace != null) return replace.toPackage() // Apply the replace directive.
```

Comment:
Nit: Consider moving the code comments to above the `if`s (and separating them with an empty line).